tests/ti.blob.addontest.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
```

Comment:
I think we can officially stop writing ES5 style code now in our test suite. Please use `let`/`const`

Reply:
I agree, but I'm going to leave this as is since this is just the _addon_ template. The test case is ES6+, which will be merge into our test suite.